panther_analysis_tool/main.py

```diff
@@ -357,21 +361,19 @@ def upload_zip(
                 else:
                     response = backend.bulk_upload(upload_params)
 
-                logging.info("Upload success.")
                 logging.info("API Response:\n%s", json.dumps(asdict(response.data), indent=4))
-
-                return_code = 0
-                return_archive_fname = ""
-                break
+                return 0, cli_output.success("Upload succeeded")
 
             except BackendError as be_err:
+                err = cli_output.multipart_error_msg(
+                    BulkUploadMultipartError.from_jsons(convert_unicode(be_err)),
+                    "Upload failed",
+                )
                 if be_err.permanent is True:
-                    logging.error("Failed to upload to backend: %s", convert_unicode(be_err))
-                    return_code = 1
-                    break
+                    return 1, f"Failed to upload to Panther: {err}"
 
                 if max_retries - retry_count > 0:
-                    logging.debug("Failed to upload to Panther: %s.", convert_unicode(be_err))
+                    logging.debug("Failed to upload to Panther: %s.", convert_unicode(err))
```

Comment:
do we need `convert_unicode()` here since we're using `err` and not `be_err`?

Reply:
Good call. Updated